scripts/generateAssetData/generateAssetData.ts

```diff
@@ -51,12 +54,12 @@ const generateAssetData = async () => {
 
   // all assets, included assets to be blacklisted
   const unfilteredAssetData: Asset[] = [
-    bitcoin,
-    bitcoincash,
-    dogecoin,
-    litecoin,
-    atom,
-    thorchain,
+    unfreeze(bitcoin),
+    unfreeze(bitcoincash),
+    unfreeze(dogecoin),
+    unfreeze(litecoin),
+    unfreeze(atom),
+    unfreeze(thorchain),
```

Comment:
blocking: from a type level, all `unfreeze()` does is remoing the `Readonly<T>` constraint, but `Readonly<Asset>[]` is actually compatible with `Asset[]`
```suggestion
    bitcoin,
    bitcoincash,
    dogecoin,
    litecoin,
    atom,
    thorchain,
```

Reply:
Yeah see here - mutation is required but we need to prevent mutating the original base assets in general
https://github.com/shapeshift/web/pull/8423#discussion_r1904771083